custom_components/tuya_local/devices/morento_mr7566_wf.yaml

```diff
@@ -0,0 +1,144 @@
+name: Air purifier
+products:
+  - id: l7eugrdimmatauv9
+    manufacturer: Morento
+    model: MR7566-WF
+entities:
+  - entity: fan
+    translation_only_key: fan_with_presets
+    icon: "mdi:air-purifier"
+    dps:
+      - id: 1
+        name: switch
+        type: boolean
+      - id: 3
+        name: preset_mode
+        type: string
+        mapping:
+          - dps_val: sleep
+            value: sleep
+          - dps_val: auto
+            value: smart
+          - dps_val: manual
+            value: normal
+      - id: 4
+        type: integer
+        name: speed
+        range:
+          min: 1
+          max: 4
+  - entity: sensor
+    class: pm25
+    dps:
+      - id: 2
+        name: sensor
+        type: integer
+        class: measurement
+        unit: ugm3
+  - entity: sensor
+    name: Filter remaining
+    category: diagnostic
+    icon: "mdi:ticket-percent-outline"
+    dps:
+      - id: 5
+        name: sensor
+        type: integer
+        unit: "%"
+  - entity: lock
+    translation_key: child_lock
+    category: config
+    dps:
+      - id: 7
+        name: lock
+        type: boolean
+  - entity: button
+    translation_key: filter_reset
+    category: config
+    dps:
+      - id: 11
+        type: boolean
+        name: button
+  - entity: select
+    translation_key: timer
+    category: config
+    dps:
+      - id: 18
+        type: string
+        name: option
+        mapping:
+          - dps_val: 0
+            value: continuous
```

Comment:
This should be "cancel" to make use of available translations.
Also when mapping strings that are numbers, it is better to enclose them in double quotes so they are read into Python as strings, not as integers.